content/03.methods.md

```diff
@@ -5,8 +5,59 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+We used simple pythonic functions called label functions to quickly annotate our candidates sentences.
+These functions are designed to emit: a positive label (1), a negative label (-1) 
+or abstain from emitting a label (0).
+We constructed label functions in the following three categories (Databases, Text Patterns and Domain Heuristics).
+These categories are described below using the following candidate sentence as an example: \"<span style="color:blue">PTK6</span> may be a novel therapeutic target for <span style="color:brown">pancreatic cancer</span>."
+
+**Databases**: This category of label functions are designed to utilize publicly available databases to determine if a co-mention pair has been previously established.
+It emits a positive label if the co-mention pair is located within a database, otherwise the given label function will abstain.
+If the label function abstains, another label function will emit a negative label.
+
+$$ \Lambda_{DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ 1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \in DB \\
+0 & otherwise \\
+\end{cases} $$
+
+$$ \Lambda_{\neg DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ -1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \notin DB \\
+0 & otherwise \\
+\end{cases} $$
+
+**Text Patterns**: This category of label functions are designed to provide contextual information such as keywords, number of words between two mentions and rules on grammatical structure of a sentence.
+The polarity of these functions can vary between positive and negative depending on the situation.
```

Comment:
```suggestion
These functions emit a positive or negative output depending on the situation.
In general, those focused on keywords emit positives and those focused on negation emit negatives.
```

Reply:
I think the second bit is true, but if it's not it should be adjusted. I thought something like this would help the reader grasp why it could be either.